adcm_aio_client/core/host_groups/_common.py

```diff
@@ -20,7 +20,7 @@
     from adcm_aio_client.core.objects.cm import Cluster, Component, Host, HostProvider, Service
 
 
-class HostsInHostGroupNode(PaginatedAccessor["Host"]):
+class HostsInHostGroupNode[ReturnObject: InteractiveObject](NonPaginatedAccessor["Host"]):
```

Comment:
`ReturnObject` is unused

`HostsInHostGroupNode(NonPaginatedAccessor["Host"])` is enough

Reply:
removed